config/opa-service/default_policies.yml

```diff
@@ -0,0 +1,16 @@
+# TODO: This will eventually be a different more fitting wac format
+"acl:Authorization":
+  - "acl:accessTo":
+      value: "default"
+      type: "*"
+    "acl:agentClass":
+      - "acl:AuthenticatedAgent"
+    "acl:mode":
+      - "acl:Read"
+  - "acl:accessTo":
+      value: "default"
+      type: "*"
```

Comment:
remove this line

Reply:
It's not technically necessary, but leaving it just for clarity for the time being